scripts/github_issue_retriever.py

```diff
@@ -44,7 +54,9 @@ def retrieve_issues(
             logger.info("Updating finished")
         else:
             logger.info("Retrieving all issues since last_modified is not available")
-            github.download_issues(owner, repo, state, retrieve_events)
+            github.download_issues(
+                owner, repo, state, retrieve_events, retrieve_private
+            )
```

Comment:
Instead of adding support for this in the `github` module, what if you add support for it as part of the github_issue_retriever script or even a separate script?
So the `github` module is "clean" and doesn't have any webcompat-specific code.

You could download all issues, then parse all of them and then download all the ones you need to download from the private repo.
Or, you could download all issues from the public and the private repos (assuming you will need almost all the ones inthe private repo anyway), and then replace the ones from the public that you need to replace with the ones from the private.

Reply:
Sounds good, I'll move it to a separate script

Reply:
> You could download all issues, then parse all of them and then download all the ones you need to download from the private repo.

Decided to go with this approach, as I only need about 1/3 of the issues from the private repo on average (for example, last week we got 655 issues and the bot closed 262 out of them, so I will only need to make a request to get content of these 262 issues from the private repo). 

I've moved the retrieval to `github_issue_retriever` script, based on `--retrieve_private` argument.